app/src/userpages/components/ProfilePage/IdentityHandler/AddIdentityButton/index.jsx

```diff
@@ -49,7 +47,7 @@ class AddIdentityButton extends React.Component<Props, State> {
                 </Button>
                 {!!modalOpen && (
                     <AddIdentityDialog
```

Comment:
You may want to wrap this one with a `<Modal>` (from shared). Otherwise the dialog renders right here.

Reply:
Ok, I'll check my other modals as well 👍